.github/workflows/e2e-test-windows.yaml

```
@@ -0,0 +1,174 @@
+          # if .head_commit.id is empty, then .head_sha is used as commit instead
+          commit_sha=${{ github.event.workflow_run.head_sha }}
+        fi
+        echo "commit_sha=${commit_sha}" >> "$GITHUB_ENV"
+
+        # Set status_context
+        status_context="ci/gh/e2e"
+        status_context="${status_context}/windows-${{matrix.windows-version}}-${{matrix.windows-featurepack}}"
+        echo "status_context=${status_context}" >> "$GITHUB_ENV"
+
+    - name: Add status to the PR check
+      run: |
```

actionlint (1.7.10)

[error] property "qe-type" is not defined in object type {}

(expression)